### PR TITLE
feat: add QGB query attestation CLI command

### PR DESCRIPTION
**Pull request by SweeXordious**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Review by rootulp — COMMENTED** [review]
Overall code LGTM but I think it would be beneficial if this PR contains tests that verify the new feature. I attempted to use it via `$ ./scripts/single-node.sh`:

```shell
$ ./build/celestia-appd query qgb attestation 1 | jq
{
  "nonce": 1,
  "members": [
    {
      "power": 4294967296,
      "evm_address": "0x966e6f22781EF6a6A82BBB4DB3df8E225DfD9488"
    }
  ],
  "height": 1,
  "time": "2023-06-07T19:50:41.120241Z"
}
$ ./build/celestia-appd query qgb attestation 2 | jq
Error: rpc error: code = Unknown desc = attestation not found: unknown request
```



Questions:
1. How would I verify that attestation 2+ work? Do I need to run something else to get more attestations added to state
2. Is that error expected for an attestation that doesn't exist yet?

**Comment by SweeXordious** [comment]
> How would I verify that attestation 2+ work? Do I need to run something else to get more attestations added to state
Is that error expected for an attestation that doesn't exist yet?

At chain start, there is only 1 attestation. And after 400 blocks, you will see another one.

For the tests, I will check if I can write something. The thing is it's just calling a query,  it's not doing anything fancy. Thus, I thought that testing it would only be testing if cobra works well, then if the query works fine (and that's already tested in the keepers)

File: x/qgb/client/query_test.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	"github.com/celestiaorg/celestia-app/test/util/network"
+	"github.com/celestiaorg/celestia-app/x/qgb/client"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	cosmosnet "github.com/cosmos/cosmos-sdk/testutil/network"
+	"github.com/stretchr/testify/suite"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+)
+
+type CLITestSuite struct {
```
[the remark this answers is not in the document]
Comment (SweeXordious) [reply x/qgb/client/query_test.go]: [67ff49e](https://github.com/celestiaorg/celestia-app/pull/1892/commits/67ff49e5d763f9c36351dc04118fa5e2c538ae11)

**Review by rootulp — APPROVED** [review]
Thanks for adding the test! LGTM. I'm not sure if you saw this:

> Is that error expected for an attestation that doesn't exist yet?

**Review by cmwaters — APPROVED** [review]

**Pull request merged**